tardis/montecarlo/montecarlo.pyx

```diff
@@ -74,7 +79,8 @@ cdef extern from "src/cmontecarlo.h":
         double inverse_sigma_thomson
         double inner_boundary_albedo
         int_type_t reflective_inner_boundary
-        double *chi_bf_tmp_partial
+        photo_xsect_1level ** photo_xsect
```

Comment:
Using double pointers can be easily avoided by using an array of structures instead of an array of   
pointers to structures. Do you think this is preferable? Since the structures are small (two pointers and an integer) this should be ok. Alternatively it is possible to store the cross section data (frequencies and cross sections) consecutively in two arrays. In this case a reference array is needed to know where the data for a certain level starts and ends.